### PR TITLE
Document live image and boot iso creation

### PR DESCRIPTION
**Pull request by M4rtinK**
Document live image and boot iso creation from the point of view
of testing changes in Anaconda, it's packaging and possibly also
related components.

Some changes can only be fully verified by building a live/boot image
and checking if it works correctly. Therefore it's important to have
a proper guide how to easily build such image that contains all the
relevant Anaconda changes.

### CODE REVIEW

**Comment by M4rtinK** [comment]
**Please not that this PR is currently WiP**

**Done**
- live image creation

**TODO**
- boot iso creation
- wait for --addrepo to be fixed

File: docs/image-creation.rst [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+    createrepo_c rpms
+
+Don't forget to rerun this when you rebuild the package, bump the version, etc.
+
+
+Setting up Mock
+---------------
+
+This guide uses a Mock chroot for both the Live image and boot iso build,
+so the this section will describe how to get Mock setup and ready for use.
```
Comment (rvykydal) [inline docs/image-creation.rst]: "the this"

File: docs/image-creation.rst [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+the Anaconda installed in the Mock chroot is used to **build** the image. If you want to test changes influencing
+live image generation you need to also install your custom Anaconda RPMs to the Mock environment.
+
+Boot iso
+--------
+
+The boot iso is basically a minimal bootable Linux distribution with just enough software to launch Anaconda and
+install an OS to a target. Unlike the Live image, which is created by Anaconda installing packages into a folder
+based on an input kickstart, boot iso is created by Lorax  without running Anaconda. Lorax has a set of
+templates, both for what to include and what to include, which are used to feed a DNF dirinstall run and then selectively
```
Comment (rvykydal) [inline docs/image-creation.rst]: "both for what to include and what to include"

File: docs/image-creation.rst [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+And that's it, you now have a live image containing your custom Anaconda package build with your changes. Note that
+the Anaconda installed in the Mock chroot is used to **build** the image. If you want to test changes influencing
+live image generation you need to also install your custom Anaconda RPMs to the Mock environment.
+
+Boot iso
+--------
+
+The boot iso is basically a minimal bootable Linux distribution with just enough software to launch Anaconda and
+install an OS to a target. Unlike the Live image, which is created by Anaconda installing packages into a folder
+based on an input kickstart, boot iso is created by Lorax  without running Anaconda. Lorax has a set of
```
Comment (jkonecny12) [inline docs/image-creation.rst]: ``created by Lorax  without running`` have two spaces between `Lorax  without`.

**Comment by VladimirSlavik** [comment]
What is needed to finish this? I'd like to see this happening.

**Comment by poncovka** [comment]
> What is needed to finish this? I'd like to see this happening.

It looks like there are some requested changes, that were not addressed yet. @M4rtinK could you look at it, please?

**Pull request closed without merging**